cinema/views.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializer import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return Response(
+                serializer.data, status=status.
+                HTTP_201_CREATED
+            )
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+def movie_detail(request, pk):
+    if request.method == "GET":
+        movie = Movie.objects.get(pk=pk)
```

Comment:
use get_object_or_404() to avoid Error

Reply:
not fixed

Reply:
but this I fixed by adding try/except or I need to fix it only with adding get_object_or_404() ? 
![image](https://github.com/mate-academy/py-movie-api/assets/111096609/9e13e3a7-8da2-42c4-aac2-0588bddd7c4e)

Reply:
yes!
it will be more better for code readability
get_object_or_404() already has try except under the hood